server/express.js

```diff
@@ -182,8 +182,9 @@ function makeApp(authAddress, cdapConfig, uiSettings) {
           },
         },
         hsts: {
-          includeSubDomains: true,
-          preload: true,
+          maxAge: cdapConfig['hsts.max.age'],
```

Comment:
we need to add these to the local development json config right? otherwise these will return undefined without the most up to date sandbox?

Reply:
Thinking about this some more:

1. There should be a main flag for turning HSTS on/off. With no flags set, HSTS should be turned off. 

```
helmet({
  strictTransportSecurity: false,
})
```

2. If the flag is on but no age is set, we should have a default age of 1 year.
3. `includeSubDomains` and `preload` can be set to `true` if the flag is on.

Reply:
the documentation website default max age is 180 days. Shall I use that value?

Reply:
```
See [Preloading Strict Transport Security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security#preloading_strict_transport_security) for details. When using preload, the max-age directive must be at least 31536000 (1 year), and the includeSubDomains directive must be present. Not part of the specification.
```

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security#directives

Reply:
HSTS preload list is a hard-coded list in Chrome; if we don't intend on submitting our code for preload, maybe we can leave the preload flag off?